### PR TITLE
Update Get-NetIPConfiguration.md

### PR DESCRIPTION
**Pull request by doctordns**
1. Added PowerShell code fencing to improve look of page
2. Removed prompt from the examples to be consistent with othere pages.

### CODE REVIEW

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR.

cc @JohanFreelancer9 for copy editing.

Thank you.

**Review by get-itips — COMMENTED** [review]
Hello @doctordns Thomas, thanks for the contribution.
I think we are including the prompt as a rule in examples, where we can, right @tseward ?

**Comment by doctordns** [comment]
The standard, I thought, was to remove the prompt. That is what happens elsewhere. 

If that is not the case, i will fix. Note that in the original there was no space between the end of the prompt and the start of the command that would need adjusting.

**Review by dariomws — COMMENTED** [review]
Thank you @doctordns.

Agree with @get-itips, I think we can leave the prompts.

**Comment by doctordns** [comment]
Updated the file adding back the prompts.

**Review by dariomws — APPROVED** [review]
Thank you so much @doctordns!

File: docset/winserver2012r2-ps/nettcpip/Get-NetIPConfiguration.md [hunk 1]
```diff
@@ -47,38 +47,38 @@ If you do not specify any parameters, this cmdlet gets IP configuration properti
 ## EXAMPLES
 
 ### Example 1: Get the IP configuration
-```
-PS C:\>Get-NetIPConfiguration
+```powershell
+PC C:\> Get-NetIPConfiguration
```
Comment (get-itips) [inline docset/winserver2012r2-ps/nettcpip/Get-NetIPConfiguration.md:51]:
```suggestion
PS C:\> Get-NetIPConfiguration
```

Comment (get-itips) [reply docset/winserver2012r2-ps/nettcpip/Get-NetIPConfiguration.md:51]: I think you meant PS here and in all of the others

**Comment by get-itips** [comment]
Please change "PC" to "PS" on the other examples thanks ! @doctordns

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by JohanFreelancer9** [comment]
Please see @get-itips' comment.

**Comment by doctordns** [comment]
> Please see @get-itips' comment.

removed the PC in favour of PS. ;-)

**Review by get-itips — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Pull request merged**